crates/sui-indexer-alt/migrations/2024-10-19-113135_ev_indices/up.sql

```diff
@@ -0,0 +1,56 @@
+CREATE TABLE IF NOT EXISTS ev_emit_mod
+(
+    package                     BYTEA,
+    module                      TEXT,
+    tx_sequence_number          BIGINT,
+    sender                      BYTEA         NOT NULL,
```

Comment:
Any reason that sender is more special than other fields? It's the only NOT NULL field.

Reply:
The other fields are implicitly not null because they are part of the primary key.